lib/Importer/Caffe2.cpp

```diff
@@ -49,6 +49,26 @@ static ArgumentDictionaryTy loadArgumentMap(const caffe2::OperatorDef &op) {
   return dict;
 }
 
+static std::vector<size_t> getPads(ArgumentDictionaryTy &dict) {
+  if (dict.count("pad")) {
```

Comment:
I tried to add "const" here, but there is compiler error: error: no viable overloaded operator[] for type 'const ArgumentDictionaryTy' (aka 'const unordered_map<basic_string<char, char_traits<char>, allocator<char> >, const onnx::AttributeProto *>')
    return getShape(dict["pads"]);
                    ~~~~^~~~~~~
In addition, I reverted getPad() function. getPad() requires all pads are equal otherwise there will be an assertion. But getPads() has no such requirement.

Reply:
The compilation error should not enforce semantics of the method.
We need to use at() operator to fix the compilation error. Operator [] cannot be used for const object, it attempts to insert default element if a search fails with the given arg in [arg].

Alternatively, you can use iterators (find instead of count) and then get an access to the field you need.

Reply:
@rdzhabarov Thanks a lot for pointing out this :)